crate/crash/repl.py

```diff
@@ -308,7 +308,7 @@ def loop(cmd, history_file):
         get_bottom_toolbar_tokens=lambda: get_toolbar_tokens(cmd),
         get_prompt_tokens=lambda: [('class:prompt', 'cr> ')]
     )
-    output = get_default_output()
+    output = create_output()
```

Comment:
Do you think it would be sensible to use `create_output(always_prefer_tty=True)`? I tried it, and the test suite will still succeed.

```python
def create_output(
    stdout: Optional[TextIO] = None, always_prefer_tty: bool = False
) -> Output:
    """
    Return an :class:`~prompt_toolkit.output.Output` instance for the command
    line.

    :param stdout: The stdout object
    :param always_prefer_tty: When set, look for `sys.stderr` if `sys.stdout`
        is not a TTY. Useful if `sys.stdout` is redirected to a file, but we
        still want user input and output on the terminal.
        By default, this is `False`. If `sys.stdout` is not a terminal (maybe
        it's redirected to a file), then a `PlainTextOutput` will be returned.
        That way, tools like `print_formatted_text` will write plain text into
        that file.
```

-- https://github.com/prompt-toolkit/python-prompt-toolkit/blob/3.0.36/src/prompt_toolkit/output/defaults.py#L19-L35

Reply:
I don't think so